templates/blog/article_list.html

```diff
@@ -1,15 +1,23 @@
 {% from 'blog/macros.html' import article_list with context %}
 {% extends 'layout.html' %}
 
-{% block title %}Nomad articles - {{ articles[0].title|truncate(40) }}{% endblock %}
+  {% block title %}Nomad articles
```

Comment:
Here you could write:

    {% block title %}Nomad articles {% if articles[0] %} - {{ articles[0].title|truncate(40) }}{% endif %}

on one line